src/linear_algebra/prod.lean

```diff
@@ -33,15 +33,18 @@ It contains theorems relating these to each other, as well as to `submodule.prod
 universes u v w x y z u' v' w' y'
 variables {R : Type u} {K : Type u'} {M : Type v} {V : Type v'} {M₂ : Type w} {V₂ : Type w'}
 variables {M₃ : Type y} {V₃ : Type y'} {M₄ : Type z} {ι : Type x}
-
+variables {M₁ N₁ N₂ N₃ : Type*}
```

Comment:
It might be better to just continue after `M₄` with `5` and `6` for consistency with the rest of the file.